server/plugins/renderer/responses/pencil-response.js

```diff
@@ -4,12 +4,7 @@ const internals = {};
 
 module.exports = function (data, assembler) {
     this.respond = function (request, reply) {
-        var response,
-            output,
-            paper,
-            templatePath;
-
-        var paper = new Paper(data.context.settings, data.context.theme_settings, assembler);
+        const paper = new Paper(data.context.settings, data.context.theme_settings, assembler, "handlebars-v4");
```

Comment:
This should be v3. We haven't yet exposed a configuration for themes to opt in to v4.

Reply:
fixed.